src/pages/404/notFound.jsx

```diff
@@ -1,7 +1,7 @@
 import React, { useEffect } from "react";
 import "./squiggly-animation.css";
 import style from "./notFound.module.css";
-import sadImg from "./../../assets/images/404/sad404.svg";
+import error1 from "./../../assets/images/404/404.gif";
```

Comment:
Rename it to something meaningful ex: `pageNotFound` or something similar.
```suggestion
import pageNotFound from "./../../assets/images/404/404.gif";
```